parsing/src/processor.ts

```diff
@@ -84,11 +91,21 @@ async function processFile(inputFile: string, outputFile: string) : Promise<{ bu
         const buildings = getPowerConsumptionForBuildings(data, producingBuildings);
 
         // Pass the producing buildings with power data to getRecipes to calculate perMin and powerPerProduct
-        const recipes = getRecipes(data, buildings);
-
+        let recipes = getProductionRecipes(data, buildings);
         removeRubbishItems(items, recipes);
         fixTurbofuel(items, recipes);
 
+        //IMPORTANT: The order here matters - don't run this because fixing the turbofuel. 
+        let powerGenerationRecipes: any[] = getPowerGeneratingRecipes(data, items);
```

Comment:
Add return type to `getPowerGeneratingRecipes` and remove it from here. Using `any` here is also very dangerous.